### PR TITLE
Updating Advanced Paste Docs

### PR DESCRIPTION
**Pull request by joadoumie**
This PR includes doc updates for the new paste actions going into Advanced Paste in the upcoming PowerToys release.

### CODE REVIEW

**Review by alvinashcraft — APPROVED** [review]
This looks good to me. When you're pasting as a txt or html file, does it prompt you for a file name or is one generated (like file1.txt)? Does that need to be mentioned somewhere in the doc?

**Comment by joadoumie** [comment]
Good catch! For now, a name with a timestamp is automatically generated.

File: hub/powertoys/advanced-paste.md [hunk 1]
```diff
@@ -100,6 +104,37 @@ Markdown output:
 > [!NOTE]
 > Paste as Markdown is a feature that runs locally and doesn't use AI.
 
+### Paste as .txt file
+
+**Paste as .txt file** enables you to paste text stored in your clipboard as a .txt file. You can optionally set a quick key shortcut in settings.
```
Comment (alvinashcraft) [inline hub/powertoys/advanced-paste.md]: Suggested wording to include the file name information. Feel free to tweak it.
```suggestion
**Paste as .txt file** enables you to paste text stored in your clipboard as a .txt file with an auto-generated file name. You can optionally set a quick key shortcut in settings.
```

File: hub/powertoys/advanced-paste.md [hunk 2]
```diff
@@ -100,6 +104,37 @@ Markdown output:
...
+Sample input: 
+
+```text
+Hello World!
+```
+If pasting files is accepted within the application that you are using (e.g. File Explorer), then the paste as .txt file action will take the input text and paste a .txt file.
+
+### Paste as .html file
+
+**Paste as .html file** enables you to paste text stored in your clipboard as a .html file. You can optionally set a quick key shortcut in settings.
```
Comment (alvinashcraft) [inline hub/powertoys/advanced-paste.md]:
```suggestion
**Paste as .html file** enables you to paste text stored in your clipboard as a .html file with an auto-generated file name. You can optionally set a quick key shortcut in settings.
```

**Comment by alvinashcraft** [comment]
I'm merging the suggestions so we can go live. We can circle back with updates later if necessary.

**Review by alvinashcraft — APPROVED** [review]

**Pull request merged**